PYME/recipes/processing.py

```diff
@@ -257,7 +257,6 @@ def calc_flow_mp(self, data, chanNum):
         return flow_x, flow_y
         
     def execute(self, namespace):
-        import multiprocessing
```

Comment:
This is not redundant! The code which uses it happens to be commented out for debugging, but it can't safely be removed.